### PR TITLE
update to scalajs 1.1.1

### PR DESCRIPTION
**Pull request by elyphas**
I am updating the library to scalajs 1.1.1
because I need it for my app.

this is one of my first tries on github.

Sorry if I am doing wrong, I will try to make it better if something bad happens.

I am not sure if the pull request was made by the way. :P

but in my local the lib is working with scalajs 1.1.1

### CODE REVIEW

File: hammerjs/src/main/scala/outwatch/libs/hammerjs/facade/Hammer.scala [hunk 1]
```diff
@@ -41,9 +41,11 @@ trait PropagatedHammer extends Hammer[PropagatingEvent] {
 // Hammer.js does not natively support event propagation. This wrapper fixes it.
 // https://github.com/hammerjs/hammer.js/issues/807
 // https://github.com/josdejong/propagating-hammerjs
-@js.native
+//@js.native
```
Comment (cornerman) [inline hammerjs/src/main/scala/outwatch/libs/hammerjs/facade/Hammer.scala:44]: Can you please remove the commented code?

File: build.sbt [hunk 2]
```diff
@@ -12,18 +11,18 @@ inThisBuild(Seq(
...
-    "scm:git:git@github.com:OutWatch/outwatch-libs.git",
-    Some("scm:git:git@github.com:OutWatch/outwatch-libs.git"))
-  )
-))
+		  scmInfo := Some(ScmInfo(
+			  url("https://github.com/OutWatch/outwatch-libs"),
+				  "scm:git:git@github.com:OutWatch/outwatch-libs.git",
+				  Some("scm:git:git@github.com:OutWatch/outwatch-libs.git"))
+			  )
+		  ))
```
Comment (cornerman) [inline build.sbt:22]: Why did you change the indent here? I think, it should stay as before.

**Comment by cornerman** [comment]
Awesome, thank you! The PR looks really good :+1: 

I just added some small comments to fix.

File: build.sbt [hunk 3]
```diff
@@ -32,15 +31,17 @@ lazy val commonSettings = Seq(
     Nil,
 
   libraryDependencies ++= Seq(
-    "org.scalatest" %%% "scalatest" % "3.1.1" % Test,
+    "org.scalatest" %%% "scalatest" % "3.2.0" % Test,
     "com.github.ghik" % "silencer-lib" % "1.6.0" % Provided cross CrossVersion.full,
-    "com.github.outwatch.outwatch" %%% "outwatch" % "f3a15f2b"
+    "com.github.outwatch.outwatch" %%% "outwatch" % "61deece"		//"f3a15f2b" original
```
Comment (cornerman) [inline build.sbt:36]: You do not need to keep the old version in the comment.

File: build.sbt (hunk 2 above)
Comment (elyphas) [reply build.sbt:22]: Sorry, It was not my intention, I was testing vim+metals at the same time and something happened I didn't realize.

File: build.sbt [hunk 4]
```diff
@@ -22,8 +21,8 @@ inThisBuild(Seq(
 ))
 
 lazy val commonSettings = Seq(
-  addCompilerPlugin("org.typelevel" %% "kind-projector" % "0.10.3"),
-  addCompilerPlugin("com.github.ghik" % "silencer-plugin" % "1.6.0" cross CrossVersion.full),
+	addCompilerPlugin("org.typelevel" %% "kind-projector" % "0.10.3"),
+	addCompilerPlugin("com.github.ghik" % "silencer-plugin" % "1.6.0" cross CrossVersion.full),
```
Comment (cornerman) [inline build.sbt:26]: There are still some extra indents here.

Comment (elyphas) [reply build.sbt]: I am sorry, I will try to take more care
